assets/css/structure.css

```diff
@@ -0,0 +1,32 @@
+.section {
+  border: var(--border);
+  margin: var(--large);
```

Comment:
As we discussed, please declare spacing variables and use them for any spaces.

Reply:
Done.